src/ucm/mmap/install.c

```diff
@@ -142,6 +132,24 @@ static ucs_status_t ucm_mmap_test(int events)
     return UCS_OK;
 }
 
+/* Called with lock held */
+static ucs_status_t ucm_mmap_test(int events)
```

Comment:
the API function should call internal helper, not the other way around..

Reply:
fixed